examples/main.tf

```diff
@@ -16,8 +16,19 @@ resource "schemaregistry_schema" "user_added" {
   schema  = file("./userAdded.avsc")
 }
 
+resource "schemaregistry_schema" "with_reference" {
+  subject = "with-reference"
+  schema = "[\"akc.test.userAdded\"]"
+
+  references {
```

Comment:
semantically it would make more sense to have a singular form for this block as in https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role#example-of-exclusive-inline-policies

Reply:
Thanks. The plural choice is coming from Hashicorps [hashicups](https://github.com/hashicorp/terraform-provider-hashicups/blob/deec90f9bd1db2c37036d63c7edf41844cef2d82/hashicups/resource_order.go#L25), but I agree, updating.